packages/gantt/backend/sirius-components-gantt/src/main/java/org/eclipse/sirius/components/gantt/description/GanttDescription.java

```diff
@@ -45,6 +45,16 @@ public record GanttDescription(String id, String label, Function<VariableManager
 
     public static final String NEW_PROGRESS = "newProgress";
 
+    public static final String SOURCE_OBJECT = "source";
+
+    public static final String TARGET_OBJECT = "target";
+
+    public static final String SOURCE_TASK = "sourceTask";
+
+    public static final String TARGET_TASKORGANTT = "targetTaskOrGantt";
```

Comment:
Does not respect snake case

Reply:
screaming snake case is used everywhere in sirius-web and in Java generally for static variables
Search for `public static final String` in sirius-web

Reply:
ok TARGET_TASKORGANTT -> TARGET_TASK_OR_GANTT